packages/serialization/multipart/src/multipartSerializationWriter.ts

```diff
@@ -30,51 +30,48 @@ export class MultipartSerializationWriter implements SerializationWriter {
 	public onBeforeObjectSerialization: ((value: Parsable) => void) | undefined;
 	public onAfterObjectSerialization: ((value: Parsable) => void) | undefined;
 	public onStartObjectSerialization: ((value: Parsable, writer: SerializationWriter) => void) | undefined;
-	public writeStringValue = (key?: string, value?: string | null): void => {
+	public writeStringValue = (key?: string, value?: string): void => {
```

Comment:
please revert the removal of null here as it does not conform with the interface